monitoring/localRun.ts

```diff
@@ -1,6 +1,44 @@
+import { prompt } from 'inquirer';
+import type { CustomScheduleEventContent } from './src';
+
+type LocalRunCLIUserInput = {
+	stage: string;
+	jurisdictions: string[];
+};
+
 async function main() {
 	const { handler } = await import('./src');
-	handler().catch(console.error);
+
+	await prompt([
+		{
+			type: 'list',
+			name: 'stage',
+			message: 'Which environment would you like to test?',
+			choices: ['code', 'prod'],
+		},
+		{
```

Comment:
I think if we run locally we never want to run against multiple regions, so in this case it should probably be a `list`.